lazycatlabs/flutter_auth_app
# feat: migrate to apimock api

## What it does

- update the library version
- change API source from `reqres.in` to `apimock.lazycatlabs.com`
- update the color scheme
- rename some files for a better understanding
- increase test coverage
- use `codecov`
- add `prd` for maestro
- update test flow in maestro
- update the app logo and image branding
- more

## How to test

1. clone the project
2. run the project
3. observe the view

## Screenshot

https://github.com/user-attachments/assets/c34dc11d-e4ba-4eb5-8db7-1df9eb881b75
